.travis.yml

```diff
@@ -33,6 +33,7 @@ matrix:
   fast_finish: true
   allow_failures:
   - env: EMBER_TRY_SCENARIO=ember-release
+  - node_js: 'stable'
```

Comment:
Should I changed it to be the same as the one in  https://github.com/ciena-frost/ember-frost-test/blob/master/.travis.yml ? But the builds are failing with Node.js: stable